dev/openbao/entrypoint.sh

```
@@ -0,0 +1,64 @@
+
+INIT_FILE="/openbao/data/.init-keys.json"
+DEV_ROOT_TOKEN="dev-root-token"
+BAO_ADDR="http://127.0.0.1:8200"
+
+# Start server in background
+bao server -config=/openbao/config/local.hcl &
+BAO_PID=$!
+
+# Wait for server to accept connections
+echo "[openbao-init] Waiting for server..."
+for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)